tests/cla.bats

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bats
+
+# SPDX-FileCopyrightText: 2021 Serokell <https://serokell.io>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+load 'test_helpers/bats-support/load'
+load 'test_helpers/bats-assert/load'
+
+@test "Without --check-localhost error should NOT be raised for broken link" {
+  run xrefcheck \
+    -c configs/github-config.yaml \
+    -r markdowns/for-cla
```

Comment:
So far, we have these two tests:

* If a config file exists with `checkLocalhost: false` and the `--check-localhost` flag was not used, then localhost links shouldn't be checked
* If a config file exists with `checkLocalhost: false` and the `--check-localhost` flag was used, then localhost links should be checked

The coverage does not seem sufficient. 
We have these variables:
* Config file may or may not exist
* If it does exist, `checkLocalhost` may be true or false
* `--check-localhost` may or may not be used

Looks like there are 6 possible combinations, so we should have 6 tests.
We're missing these 4 tests:

* If a config file does not exist and the `--check-localhost` flag was not used
* If a config file does not exist and the `--check-localhost` flag was used
* If a config file exists with `checkLocalhost: true`, and the `--check-localhost` flag was not used
* If a config file exists with `checkLocalhost: true`, and the `--check-localhost` flag was used